registry/sql-registry/registry/db_registry.py

```diff
@@ -6,11 +6,40 @@
 from registry import connect
```

Comment:
Could you add some test cases for code changes under registry/sql-registry in registry/test/test_sql_registry.py